scripts/convert_hf_checkpoint.py

```diff
@@ -120,16 +120,19 @@ def copy_weights_hf_llama(
     weight_map = {
         "model.embed_tokens.weight": "transformer.wte.weight",
         "model.layers.{}.input_layernorm.weight": "transformer.h.{}.norm_1.weight",
+        "model.layers.{}.input_layernorm.bias": "transformer.h.{}.norm_1.bias",
```

Comment:
Why is bias necessary here if the config sets `bias=False`?

Reply:
Config's `bias` parameter doesn't affect normalization layer.
`Zephyr` uses `torch.nn.LayerNorm`, which by default sets `bias=True`.
So that's the reason.

It adds some level of discrepancy, since for `RMSNorm` class it's not possible to have bias at all.